buildSrc/src/main/kotlin/com/getstream/sdk/chat/Dependencies.kt

```diff
@@ -28,7 +28,7 @@ private const val CONSTRAINT_LAYOUT_VERSION = "2.0.3"
 private const val COROUTINES_VERSION = "1.4.1"
 private const val DEXTER_VERSION = "6.2.1"
 private const val DRAWABLETOOLBOX_VERSION = "1.0.7"
-private const val DOKKA_VERSION = "1.4.20"
+private const val DOKKA_VERSION = "1.4.20.2-dev-62"
```

Comment:
Isn't it working on a stable version yet?

Reply:
Not on stable, not even on RC 😅. Only in the development versions for now.